### PR TITLE
various fixes for 5.0.18

### PR DESCRIPTION
**Pull request by grahamegrieve**

### CODE REVIEW

**Comment by grahamegrieve** [comment]
@markiantorno review and release please

File: org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/DiagnosticReportRenderer.java [hunk 1]
```diff
@@ -218,23 +372,113 @@ private void addObservationToTable(XhtmlNode tr, ResourceWrapper obs, int i) {
...
+      pw = getProperty(obs, "issued");
+      if (valued(pw)) {
+        if (!Base.compareDeep(pw.value().getBase(), eff, true)) {
+          render(td, pw.value());
+        }
+      }
+    }
   }
-
 }
```
Comment (markiantorno) [inline org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/DiagnosticReportRenderer.java:484]: Is there any plan to localize the Strings in this file?

File: org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/utils/Resolver.java [hunk 2]
```diff
@@ -3,46 +3,58 @@
...
+      this.containerElement = bundle;
       this.resourceElement = dr;
     }
 
-    public ResourceContext(Object bundle, Element doc) {
-      // TODO Auto-generated constructor stub
-    }
+//    public ResourceContext(Object bundle, Element doc) {
+//      // TODO Auto-generated constructor stub
+//    }
```
Comment (markiantorno) [inline org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/utils/Resolver.java:48]: We should just remove the commented code if we're not using it.

File: org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/utils/Resolver.java [hunk 3]
```diff
@@ -3,46 +3,58 @@
...
 
-    public Bundle getBundleResource() {
-      return bundleResource;
-    }
+//    public Bundle getBundleResource() {
+//      return containerResource;
+//    }
```
Comment (markiantorno) [inline org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/utils/Resolver.java:51]: We should just remove the commented code if we're not using it.

File: org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/utils/Resolver.java [hunk 4]
```diff
@@ -3,46 +3,58 @@
...
 
-    public org.hl7.fhir.r5.elementmodel.Element getBundleElement() {
-      return bundleElement;
-    }
 
+    //    public org.hl7.fhir.r5.elementmodel.Element getBundleElement() {
+//      return containerElement;
+//    }
+//
```
Comment (markiantorno) [inline org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/utils/Resolver.java:57]: We should just remove the commented code if we're not using it.

**Pull request merged**